lib/puppet/provider/jail/pyiocage.rb

```diff
@@ -6,28 +6,42 @@
   defaultfor kernel: :freebsd
 
   # this is used for further confinement
-  commands iocage: '/usr/local/bin/iocage'
+  commands pyiocage: '/usr/local/bin/iocage'
 
-  def self.pyiocage(*args)
+  def self.iocage(*args)
     cmd = ['/usr/local/bin/iocage', args].flatten.join(' ')
     execute(cmd, override_locale: false)
```

Comment:
as soon as https://github.com/iocage/iocage/issues/259 is fixed, we have to turn on `failonfail: true` here.